diff --git a/node_modules/youtube-chat/dist/parser.js b/node_modules/youtube-chat/dist/parser.js
index c92f421..984fe65 100644
--- a/node_modules/youtube-chat/dist/parser.js
+++ b/node_modules/youtube-chat/dist/parser.js
@@ -94,7 +94,7 @@ function parseMessages(runs) {
             // Emoji
             const thumbnail = run.emoji.image.thumbnails.shift();
             const isCustomEmoji = Boolean(run.emoji.isCustomEmoji);
-            const shortcut = run.emoji.shortcuts[0];
+            const shortcut = (run.emoji?.shortcuts?.length > 0) ? run.emoji?.shortcuts[0] : "";
             return {
                 url: thumbnail ? thumbnail.url : "",
                 alt: shortcut,
